source/_lovelace/button.markdown

```diff
@@ -7,10 +7,35 @@ description: "The Button card allows you to add buttons to perform tasks."
 The Button card allows you to add buttons to perform tasks.
 
 <p class='img'>
-<img src='/images/lovelace/lovelace_entity_button_card.png' alt='Screenshot of the button card'>
-Screenshot of the Button card.
+<img src='/images/lovelace/lovelace_entity_button_card.png' alt='Screenshot of the button cards'>
+Screenshot of three Button Cards.
 </p>
 
+## Card Settings
+
+{% configuration_basic %}
+Entity:
+  description: The entity the card interacts with, for example `light.living_room`.
+Name:
+  description: The button name. The default is the name of the entity, if an entity is selected.
+Icon:
+  description: The icon shown on the card. The default is the entity domain icon, if an entity is selected.
+Show Name?:
+  description: A toggle to show or hide the button name.
+Show Icon?:
+  description: A toggle to show or hide the icon.
+Icon Height:
+  description: The height of the icon, in pixels.
```

Comment:
```suggestion
  description: The height of the icon, in pixels. (Ex. 50px)
```

Reply:
If I’m adding a new card, or editing an existing one, the editor already shows me that whatever I enter will be in pixels, so even that is almost redundant information. As soon as I enter a number, the icon height changes in the card preview. I’m not sure we’re adding much useful information by giving an example.

Reply:
Good point 👍